src/extension.ts

```diff
@@ -62,6 +53,27 @@ export function activate(context: vscode.ExtensionContext) {
 	}
 }
 
+function createScrapbook(): void {
+	let uri: vscode.Uri = null;
+	let count = 1;
+	const max = 99999;
+	while (count < max) {
+		uri = vscode.Uri.file(path.join(vscode.workspace.rootPath, `Scrapbook-${count}.mongo`));
+		if (!vscode.workspace.textDocuments.find(doc => doc.uri.fsPath === uri.fsPath) && !fs.existsSync(uri.fsPath)) {
```

Comment:
Checking file existence could be slow if the workspace path is not local. Consider making the entire function async?